learning-development/r.qmd

```diff
@@ -92,15 +92,15 @@ Sometimes renv::restore() can fail, and when in specific renv-controlled project
 
 1. Configuring the proxy settings by running the below in R - this also helps if you are getting timeout issues when trying to webscrape with R:
 
-```
+``` {r proxy_settings, eval=FALSE}
 Sys.setenv(no_proxy="*") 
 
 ```
 
 
 2. Specifying the renv library as the install location. It's a bit of a fudge, though these lines are helpful to get the packages from the renv lockfile installed and you running the project when needed:
-
-```
+ 
```

Comment:
Looks like stray whitespace?

Reply:
I don't even remember editing this line 🤣